app/helpers/application_helper.rb

```diff
@@ -24,6 +24,44 @@ def miq_accordion_panel(title, condition, id, &block)
     end
   end
 
+  def single_relationship_link(record, table_name, property_name = nil)
+    out = ''
+    property_name ||= table_name
+    ent = record.send(property_name)
+    name = ui_lookup(:table => table_name.to_s)
+    if role_allows(:feature => "#{table_name}_show") && !ent.nil?
+      out = content_tag(:li) do
+        link_to("#{name}: #{ent.name}",
+                {:controller => table_name, :action => 'show', :id => ent.id.to_s},
+                :title       => _("Show this %{entity_name}'s parent %{linked_entity_name}") %
+                                {:entity_name        => record.class.name.demodulize.titleize,
```

Comment:
not sure about this `record.class.name.demodulize.titleize` for the i18n, @mzazrivec: should we replace that with some `ui_lookup` call?